### PR TITLE
added doc - more in standard algorithms, update index, new summary images for routing spec and computation

### PR DESCRIPTION
**Pull request by zamusacelre**

### CODE REVIEW

File: docs/source/involved-opendcs.rst [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+
+**************************
+GitHub Repository Overview
+**************************
+
+
+opendcs
+=======
+
```
Comment (MikeNeilson) [inline docs/source/involved-opendcs.rst:63]: This is the "Organization" URL, the URL for OpenDCS (the core project that most interact with) is https://github.com/opendcs/opendcs

**Review by MikeNeilson — COMMENTED** [review]
Looks great. But the Repository Overview section has a lot of new lines and I think you were still working on that.

File: docs/source/involved-opendcs.rst [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+.. image:: ./media/involved/opendcs/im-03-freelists-opendcs-dev-archive.JPG
+   :alt: archive free lists opendcs dev
+   :width: 550
+
+
+**************************
+GitHub Repository Overview
+**************************
+
+
```
Comment (ktarbet) [inline docs/source/involved-opendcs.rst:60]:
```suggestion

The OpenDCS organization URL is :  https://github.com/opendcs/opendcs

Please use GitHub for Discussions, reporting issues, and contributions using pull-requests.

```

File: docs/source/involved-opendcs.rst [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+
+
+
+
+
+
+
```
Comment (ktarbet) [inline docs/source/involved-opendcs.rst:77]:
```suggestion

```

**Review by ktarbet — APPROVED** [review]

**Review by MikeNeilson — APPROVED** [review]

**Pull request merged**